prolog/sparqlprog/ontologies/allie_sflf.pl

```diff
@@ -0,0 +1,28 @@
+:- module(allie_sflf,
+ [
+    in_cluster/2,
+    get_longRef/2,
+    get_longForm/2,
+    get_shortForm/2,
+    get_longForms/4
+ ]).
+
+:- sparql_endpoint(allie, 'https://data.allie.dbcls.jp/sparql').
+
+:- rdf_register_prefix(allie, 'http://purl.org/allie/ontology/201108#').
+
+in_cluster(A, B) :- rdf_path(A, ( (allie:contains) / (allie:hasMemberOf) ), B).
+get_longRef(A, B) :- rdf(A, allie:hasLongFormRepresentationOf, B).
+get_longForm(A, B) :- rdf(A, allie:hasLongFormOf, B).
+get_shortForm(A, B) :- rdf(A, allie:hasShortFormOf, B).
+
+get_longForms(S, L1, L2, LANG) :-
```

Comment:
In logic programming it's customary not to give procedural names such as get_X to predicates. Think of predicates as relations, or tables in a relational database. Also use of plurals is discouraged unless the argument is a list. Maybe one of:

 - has_long form
 - abbreviation_has_long_from

(also it's customary in prolog to use snake_case over upperCase)

Reply:
Thanks. I changed names. Are these fine?